usaspending_api/matviews/migrations/0001_initial.py

```diff
@@ -0,0 +1,1142 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-01-07 12:53
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+import django.contrib.postgres.search
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('awards', '0070_move_matview_models'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AwardSearchView',
+            fields=[
+                ('keyword_ts_vector', django.contrib.postgres.search.SearchVectorField()),
```

Comment:
Since these are all unmanaged, is there a way to not include all several hundred lines of fields and define an empty model for the initial migration? Might just be a pet peeve of mine